src/guidelines.rst

```diff
@@ -70,8 +70,4 @@ Fortunately, there are options for optimizing the development of a package.
 Renaming Packages
 =================
 
-Sometimes packages are misnamed. To correct the name of the package, just
-change the name of the package in the ``recipe.yaml`` and rerender with
-``conda smithy rerender``. Note that this will not change the name of the repo
-and that the package with the old name will still exist. Please contact a
-member of conda-forge/core to remove the old package.
+Sometimes packages are misnamed. To correct the name of the package, please submit a PR into staged-recipes with the correct name. During the review process please make certain to note that the package is a rename and contact a member of conda-forge/core to remove the old package.
```

Comment:
You're definitely right that the feedstock needs to get removed. 

I think my question is what should we do with the existing package? Anyone who has created a package that depends on the "old" package before the rename will have their builds break underneath them if we remove the package from anaconda.org/conda-forge/<old_package_name>. As a rule, I am opposed to unexpectedly breaking downstream builds. 

As far as conda-forge policy on what to do with the "legacy" packages, I think that we need to decide on whether to leave or remove the legacy package on a case-by-case basis. Maybe we could just update the language to be 

"contact a member of conda-forge/core if you wish to remove the old package"

That way the decision on what to do with the old artifact resides with the person/team that wishes to do the rename. Thoughts?